Shared_UI/Shared_UI/Extension/UITextField + Extension.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  UITextField + Extension.swift
+//  Shared_UI
+//
+//  Created by Ahmed Atef on 26/03/2023.
+//
+
+import Foundation
+import UIKit
+
+extension UITextField {
+    public func applyPaswwordTextField() {
+        isSecureTextEntry = true
+        rightView = createToggleButton()
+        rightViewMode = .always
+    }
+    
+    private func createToggleButton() -> UIButton {
+        let eyeButton = UIButton()
+        eyeButton.setImage(.hidePassswordIcon, for: .normal)
+        eyeButton.tintColor = .black
+        if #available(iOS 15.0, *) {
+            var eyeButtonConfiguration = UIButton.Configuration.plain()
+            eyeButtonConfiguration.buttonSize = .small
+            eyeButton.configuration = eyeButtonConfiguration
+        } else {
+            eyeButton.imageEdgeInsets = UIEdgeInsets(top: 0, left: -8, bottom: 0, right: 8)
```

Comment:
why those hard-coded value like 8 , -8

Reply:
8 to give space from the right and if made 0 from left the size of the button will decrease

Reply:
did got it

Reply:
got what?